rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/CovidImageViewFactoryTest.java

```diff
@@ -19,21 +25,31 @@
                 }}
         ));
+    }
+
+    @Test
+    public void testGetViewsFromJson() throws Exception {
+        List<View> views = covidImageViewFactory.getViewsFromJson("step1", RDTApplication.getInstance(), null, new JSONObject(), null);
+        Assert.assertEquals(1, views.size());
+        ImageView imageView = views.get(0).findViewById(R.id.image);
+        imageView.setImageBitmap(ReflectionHelpers.callInstanceMethod(covidImageViewFactory, "getBitmap",
```

Comment:
Shouldn't the image be set automatically. Why do we need to set it here?

Reply:
fixed.